src/api/mod.rs

```diff
@@ -8,20 +8,20 @@
 
 mod auth;
 mod constants;
-mod errors;
+pub mod errors;
```

Comment:
You don't make the whole mod public, only the things we consider part of the public API, the rest should be private to the api crate mods

Reply:
I pushed a change to this file to bindings branch, you shouldn't need any change to this file once you rebase with it